benchmarks/GeneratedHydratorBenchmark/AllPrivateClassHydrationBench.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace GeneratedHydratorBenchmark;
+
+/**
+ * @BeforeMethods({"setUp"})
+ */
+class AllPrivateClassHydrationBench extends AbstractHydrationBench
+{
+    public function setUp()
+    {
+        $this->createHydrator(AllPrivateClass::class);
+        $this->createData();
+    }
+
+    /**
+     * @Revs(100)
+     * @Iterations(200)
+     */
+    public function benchConsume()
+    {
+        $object = new AllPrivateClass();
```

Comment:
Actually the _right way_(TM) would be to use data sets/source/provider (don't know the name they give it in phpbench) just moving it to the constructor right now, I will explore this later.